INSTALL.md

```diff
@@ -64,6 +67,9 @@ filter-repo only consists of a few files that need to be installed:
     displaying the html version of the help, when help.format is set to
     "html" (the default on Windows).
 
+    If you've cloned the repo directly `make snag_docs` must be run to
```

Comment:
same -- "can" instead of "must".

(Folks can build the docs manually, even if not documented how to do so.  They could also check them out from the other branch or download them from online, or just ignore these documentation files.)